configure.ac

```diff
@@ -53,7 +53,17 @@ m4_foreach_w([_m], [
 # then some tests will be skipped, but no tests should fail. Only
 # prints a warning if not installed.
 m4_foreach_w([_m], [
+    Carp
+    Exporter
+    HTTP::Daemon=6.12
+    HTTP::Daemon::SSL
+    HTTP::Message::PSGI
+    IO::Socket::IP
+    Scalar::Util
+    Test::TCP
     Test::Warnings
+    Time::HiRes
+    URI
```

Comment:
Can we move this to error? This makes getting the other tests complete easier.

Reply:
Modules used only by tests shouldn't cause `configure` to fail because the user isn't required to run `make check` to build and install ddclient.

Reply:
Yeah, you are right. Missed that.